lib/tapioca/dsl/compilers/active_record_columns.rb

```diff
@@ -153,12 +153,18 @@ def gather_constants
             methods_to_add: T.nilable(T::Array[String]),
             return_type: String,
             parameters: T::Array[[String, String]],
+            kw_parameters: T::Array[[String, String, String]],
           ).void
         end
-        def add_method(klass, name, methods_to_add, return_type: "void", parameters: [])
+        def add_method(klass, name, methods_to_add, return_type: "void", parameters: [], kw_parameters: [])
```

Comment:
Can we instead make the `parameters` parameter take a `T::Array[RBI::Param]` instead? I see just a single use of this method that supplies a parameter already, and it can easily supply `create_param(value, type: setter_type)` instead.

That way, there wouldn't be any split or extra processing for kw parameters.

Reply:
Sure, only minor tweak is that it has to be a `T::Array[RBI::TypedParam]`, I'm not sure if that would be wildly/unexpectedly different from what you expected

Reply:
Yeap, that's exactly what I meant to write actually. :) Thanks for doing the change.